ext/rbtrace.c

```diff
@@ -79,7 +79,7 @@ timeofday_usec()
 #define MAX_TRACERS 100 // max method tracers
 #define MAX_EXPRS 10    // max expressions per tracer
 #ifndef BUF_SIZE        // msgq buffer size
-#define BUF_SIZE 120
+#define BUF_SIZE 1024
```

Comment:
120B wasn't enough for the script in `heapdump`.

Reply:
yeah I feel like this limit was quite annoying in the past